CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+All repositories in these organizations:
+
+* [lightbend](https://github.com/lightbend)
+* [akka](https://github.com/akka)
+* [lagom](https://github.com/lagom)
+* [playframework](https://github.com/playframework)
+* [sbt](https://github.com/sbt)
+* [slick](https://github.com/slick)
+
+are covered by the Lightbend Community Code of Conduct: https://www.lightbend.com/conduct
```

Comment:
https://github.com/lightbend/mima/community already picks up the one from https://github.com/lightbend/.github - perhaps that's sufficient?

Reply:
I don't have a strong very feeling about it one way or the other, but I rather like having the CoC show up in local clones and not just in the GitHub UI.